IndexedDB/idbcursor-request-source.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>IndexedDB: The source of requests made against cursors</title>
+<meta name="help" href="https://w3c.github.io/IndexedDB/#dom-idbrequest-source">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+<script>
+
+[
+  'cursor.update(0)',
+  'cursor.delete()'
+].forEach(expr => indexeddb_test(
+  (t, db) => {
+    db.createObjectStore('store', {autoIncrement: true});
+  },
+  (t, db) => {
+    const tx = db.transaction('store', 'readwrite');
+    const store = tx.objectStore('store');
+    store.put('value');
+    store.openCursor().onsuccess = t.step_func(e => {
+      const cursor = e.target.result;
+      assert_equals(eval(expr).source, cursor,
```

Comment:
nit: You could avoid eval here by having each string be a closure instead, and using toString() on the closure. To be clear, here's some incomplete code below.

```js
[
  cursor => { cursor.update(0) },
].forEach(fn ...

assert_equals(fn(cursor), cursor, `${fn}`.source ... })
```

Same for the cases below.

Reply:
Done.